### ISSUE TITLE
FileInput - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#4813]
Add Accessibility information to the FileInput page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the FileInput component and if they are passing/failing.
- [ ] Create a pull request on the Design System site to add this information to the FileInput page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/4675/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [ ] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

**Comment by jcfilben** [comment]
Thanks Cillian! I went through and added some comments to the document

**Comment by CillianHPE** [comment]
@jcfilben 

Created an **[issue](https://github.com/grommet/hpe-design-system/issues/4879)** in the backlog for the FileInput a11y issues

**Comment by jcfilben** [comment]
Follow on issues created:
https://github.com/grommet/grommet/issues/7563
https://github.com/grommet/grommet/issues/7562
https://github.com/grommet/grommet/issues/7561

Marking this as closed
